static/app/views/performance/transactionSummary/utils.tsx

```diff
@@ -133,6 +135,30 @@ export function generateTransactionLink(transactionName: string) {
   };
 }
 
+export function generateReplayLink(routes: PlainRoute<any>[]) {
+  return (
+    organization: Organization,
+    tableRow: TableDataRow,
+    _query: Query
+  ): LocationDescriptor => {
+    const replayId = tableRow.replayId;
+    const replaySlug = `${tableRow['project.name']}:${replayId}`;
+    const referrer = encodeURIComponent(getRouteStringFromRoutes(routes));
+
+    const transactionTimestamp = tableRow.timestamp
```

Comment:
I'm using here the timestamp that the row is displaying, but I don't seem to get the exact initial time of the transaction

Reply:
In this [example](https://localhost:7999/organizations/sentry-emerging-tech/replays/sourcemapsio-replays:aac938ca8a8d4162a20e986b7724e2bc/?event_t=1665441626000&referrer=%252Forganizations%252F%253AorgId%252Fperformance%252Fsummary%252F&t_main=trace) the initial time offset is a bit ahead from the pageload event, which in my undestanding that should be where the transaction really started

![image](https://user-images.githubusercontent.com/39612839/195195729-b4c979a5-a099-47d6-84c5-762bc0e3acf3.png)

Reply:
how far is 'a bit ahead' what are the numbers?